### PR TITLE
jpwr integration

### PR DESCRIPTION
**Pull request by chelseajohn**
Integrate jpwr workflow via JUBE into CARAML

### CODE REVIEW

**Comment by chelseajohn** [comment]
ResNet50+jpwr  does not work smoothly for all devices. 
The following issues were observed with jpwr :
- not compatible with python <3.9
- gets stuck on GH200 and JEDI systems
-  does not produce power files for all ranks always
- does not work with NVIDIA systems on more than one node

@stepannassyr is the combined energy csv file supposed to show only results with rank0 for each node?

**Comment by stepannassyr** [comment]
- GH200/Jedi: Does it get stuck or does it run into the PyNVML bug? This one is inconsistent (works sometimes, sometimes it doesn't). If it's a separate issue, I'll look into that
- python <3.9: I thought this was IPU only, so I just removed the annotations in the ipu file, I'll do that for the others as well
- Power files for all ranks: i.e. the combination step fails because it can't find the files?
- more than one node NVIDIA: I had success running this on multiple NVIDIA nodes, is it the PyNVML error?

File: resnet50/resnet50_benchmark.xml [hunk 1]
```diff
@@ -286,22 +285,29 @@
...
-      <use>modelParameter</use>
-      <use>environment</use>
-      <use>executeset</use>
-      <use from="platform.xml">jobfiles</use>
-      <use from="platform.xml">executesub</use>
-      <do>$submit $submit_script</do>
-      <do done_file="$done_file"></do>
-    </step>
+      <do>module load GCC SciPy-bundle</do>
+      <do>python3 ${jube_benchmark_home}/aux/combine_energy.py --output ${jube_benchmark_home}/${jube_wp_relpath}/combined_energy.csv $$(printf "${jube_benchmark_home}/${jube_wp_relpath}/execute/${system_name}_power.%d.energy.csv " $$(seq 0 $$(($nodes-1))))</do>
```
Comment (stepannassyr) [inline resnet50/resnet50_benchmark.xml:309]: Ah, `$nodes` should be replaced with number of ranks instead, same for llm actually

Comment (stepannassyr) [reply resnet50/resnet50_benchmark.xml:309]: in this expression: `$$(seq 0 $$(($nodes-1))))`

**Review by stepannassyr — APPROVED** [review]

**Pull request merged**